protos/sigstore_bundle.proto

```diff
@@ -57,9 +57,38 @@ message TimestampVerificationData {
 // the key identifier, it MUST match the `keyid` field of the signature the
 // extension is attached to.
 message VerificationMaterial {
+        // The key material for verification purposes.
+        //
+        // This allows key material to be conveyed in one of three forms:
+        //
+        // 1. An unspecified public key identifier, for retrieving a key
+        //    from an out-of-band mechanism (such as a keyring);
+        //
+        // 2. A sequence of one or more X.509 certificates, of which the first member
+        //    MUST be a leaf certificate conveying the signing key.
+        //
+        //    Signers MUST NOT include root CA certificates in bundles, and SHOULD NOT
+        //    include intermediate CA certificates that appear in an independent root of trust
+        //    (such as the Public Good Instance's trusted root).
+        //
+        //    Verifiers MUST validate the chain carefully to ensure that it chains up
+        //    to a CA certificate that they independently trust. Verifiers SHOULD
+        //    handle old or non-complying bundles that have additional intermediate and/or
+        //    root CA certificates.
+        //
+        // 3. A single X.509 certificate, which MUST be a leaf certificate conveying
+        //    the signing key.
+        //
+        // When used with the Public Good Instance (PGI) of Sigstore, form (1) MUST NOT be used,
```

Comment:
Form 1 should be allowed for self-managed keys. You can use the public instance of Rekor without the public instance of Fulcio. I'm assuming this comment is about using Fulcio? Is there a way we can convey that if so?

Reply:
Yeah, I'll clarify that this is about using Fulcio/"keyless" flows.